tools/scripts/cargo_raze.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euxo pipefail
+cd "$(git rev-parse --show-toplevel)"
+(
+cd third_party/cargo && cargo raze
+)
+./tools/scripts/format_build_files.sh
```

Comment:
have you documented this script in a README somewhere?

i'd love to have a description of when and how to update rubyfmt's deps etc.

Reply:
I'll write a readme for this